test/controllers/FunctionalTest.java

```diff
@@ -8,38 +10,38 @@
 import org.junit.Test;
 import play.test.TestServer;
 
+import java.util.Collections;
+import java.util.concurrent.ArrayBlockingQueue;
 import java.util.concurrent.CompletableFuture;
 
-import static com.jayway.awaitility.Awaitility.await;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.fail;
 import static play.test.Helpers.running;
 import static play.test.Helpers.testServer;
+import static org.awaitility.Awaitility.*;
 
 public class FunctionalTest {
 
     @Test
     public void testRejectWebSocket() {
-        TestServer server = testServer(31337);
+        TestServer server = testServer(37117);
         running(server, () -> {
             try {
                 AsyncHttpClientConfig config = new DefaultAsyncHttpClientConfig.Builder().setMaxRequestRetry(0).build();
                 AsyncHttpClient client = new DefaultAsyncHttpClient(config);
                 WebSocketClient webSocketClient = new WebSocketClient(client);
 
                 try {
-                    // localhost:31337 is not an acceptable origin to the server, so this will fail...
-                    String serverURL = "ws://localhost:31337/ws";
-                    WebSocketClient.LoggingListener listener = new WebSocketClient.LoggingListener();
-                    CompletableFuture<WebSocket> completionStage = webSocketClient.call(serverURL, listener);
-
-                    await().until(() -> {
-                        assertThat(completionStage)
-                                .hasFailedWithThrowableThat()
-                                .hasMessageContaining("Invalid Status Code 403");
+                    String serverURL = "ws://localhost:37117/ws";
+                    WebSocketClient.LoggingListener listener = new WebSocketClient.LoggingListener(message -> {
+                        // System.out.println(message);
```

Comment:
Done.  BTW, I lost the ability to merge PRs on the examples somewhere down the line (github status altered?)